src/pages/List/List.js

```diff
@@ -1,106 +1,15 @@
 import React, { useEffect, useState } from 'react';
-import { useNavigate, useLocation } from 'react-router-dom';
-import LikeButton from '../../components/LikeButton/likeButton';
+import { useLocation } from 'react-router-dom';
 import Topnav from '../../components/Topnav/Topnav';
 import Footer from '../../components/Footer/Footer';
 import './List.scss';
-
-function ProductCard({ src1, src2, src3, productId, colorId }) {
-	const navigate = useNavigate();
-	function handleClick() {
-		navigate(`/products/${productId}?color=${colorId}`);
-	}
-
-	return (
-		<div className="imageFull">
-			<div className="imageContainer1" onClick={handleClick}></div>
-			<img src={src1} className="img1" alt="상품" width="300px" height="300px" />
-
-			<div div className="imageContainer2" onClick={handleClick}></div>
-			<img src={src2} className="img2" alt="상품" width="300px" height="300px" />
-
-			<div className="imageContainer3" onClick={handleClick}></div>
-			<img src={src3} className="img3" alt="상품" width="300px" height="300px" />
-		</div>
-	);
-}
-
-function ProductCard2({ src1, src2, number, index, setNumber }) {
-	const clickImg1 = () => {
-		const arr = [...number];
-		arr[index] = 0;
-		setNumber(arr);
-		console.log(0);
-	};
-
-	const clickImg2 = () => {
-		const arr = [...number];
-		arr[index] = 1;
-		setNumber(arr);
-	};
-
-	return (
-		<div className="subImages">
-			<img src={src1} alt="상품" width={50} height={50} onClick={clickImg1} />
-			<img src={src2} alt="상품" width={50} height={50} onClick={clickImg2} />
-		</div>
-	);
-}
-
-function Sortbox({ className, category }) {
-	const navigate = useNavigate();
-	function handleClick1() {
-		navigate(`/products`);
-	}
-	function handleClick2() {
-		navigate(`?category=${category}&sort=discount_price-asc`);
-	}
-	function handleClick3() {
-		navigate(`?category=${category}&sort=discount_price-desc`);
-	}
-	return (
-		<form className={className}>
-			<div className="sort">SORT BY</div>
-			<br />
-			<div onClick={handleClick1}>
-				- <span className="sortName">Recent</span>
-			</div>
-			<br />
-			<div onClick={handleClick2}>
-				- <span className="sortName">Price (Low)</span>
-			</div>
-			<br />
-			<div onClick={handleClick3}>
-				- <span className="sortName">Price (High)</span>
-			</div>
-		</form>
-	);
-}
-
-function ProductCardMain({ content, price, newprice, productId, colorId }) {
-	const navigate = useNavigate();
-	function handleClick() {
-		navigate(`/products/${productId}?color=${colorId}`);
-	}
-	return (
-		<div className="contentContainer" onClick={handleClick}>
-			<h3>{content}</h3>
-			<div className="priceContainer">
-				<span className="price">￦{price}</span>
-				<span className="newprice">￦{newprice}</span>
-			</div>
-		</div>
-	);
-}
+import ProductCardWrapper from './ProductCardWrapper';
+import SortBox from './SortBox';
 
 function List() {
 	const [productList, setProductList] = useState([]);
 	const [checked1, ischecked1] = useState('sortBox1');
 
```

Comment:
```suggestion
	const [checked, ischecked] = useState('sortBox');
```

변수명이나 클래스명을 숫자로 나누는 것은 가독성이 좋지 않습니다. 리팩토링을 하셔서 이제 숫자로 나눌 필요가 없어보이는데 수정하면 좋을 것 같습니다!

Reply:
네 이건 사실 클래스 명이라서 저렇게 해둔건데 이번에는 명시적으로 바꿨고 숫자 전부 삭제했습니다.